src/explore-education-statistics-frontend/src/modules/data-catalogue/DataCataloguePageNew.tsx

```diff
@@ -261,6 +262,19 @@ export default function DataCataloguePageNew() {
           : undefined
       }
     >
+      <NotificationBanner title="This page has changed">
```

Comment:
How closely do we need to fit the design system? This is ludicrously niggly but I couldn't help but notice it asks us to "Position a notification banner immediately before the page h1.", and it looks like the <Page> component renders children just below its h1. I definitely don't think it matters though, just noticed it and thought I'd flag it

Reply:
Hmm good spot! The component has a h2 in so it would seem odd to me to have it above the h1. We're slightly misusing it here anyway and it's only temporary so I think it's probably ok.